charts/launch-agent/templates/namespace.yaml

```diff
@@ -1,9 +1,11 @@
+---
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: wandb
+  name: {{ .Values.namespace }}
   labels:
     pod-security.kubernetes.io/enforce: baseline
     pod-security.kubernetes.io/enforce-version: latest
     pod-security.kubernetes.io/warn: baseline
-    pod-security.kubernetes.io/warn-version: latest
\ No newline at end of file
+    pod-security.kubernetes.io/warn-version: latest
+...
```

Comment:
what does ... mean here?

Reply:
Does this need an {{- end}}?

Reply:
... means it is the end of that yaml section.

Reply:
@bcsherma, should we wrap this?

Reply:
Not sure what you mean exactly?